scipy/optimize/_optimize.py

```diff
@@ -41,9 +41,6 @@
 from scipy._lib._util import MapWrapper, check_random_state
 from scipy.optimize._differentiable_functions import ScalarFunction, FD_METHODS
 
-# deprecated imports to be removed in SciPy 1.13.0
-from numpy import asfarray  # noqa
```

Comment:
Can you remove `asfarray` from here too:
https://github.com/scipy/scipy/blob/5e136d908a50f18a7a99d0d6645d5cae957dc737/scipy/optimize/optimize.py#L22

Reply:
Oops sorry I see you already have